app/controllers/vacancies_controller.rb

```diff
@@ -47,6 +47,14 @@ def destroy
     redirect_to vacancies_path
   end
 
+  def my_vacancies
+    if current_user.nil?
```

Comment:
for this we use method's devise gem, before_action :authenticate_user!, only: []

Reply:
done